scripts/test-example-kafka.json

```diff
@@ -4,8 +4,9 @@
   "msgsPerSecond": 100000,
   "msgSizeBytes": 1024,
   "batchSizeSend": 1000,
+  "senderConcurrency": 4,
```

Comment:
as I mentioned for Kafka we should have much higher sender concurrency - so that we can fill in the whole buffer at least once which is supposed to be sent

Reply:
I changed it to 200 = 2*msgsPerSecond/batchSizeSend